Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -708,53 +714,35 @@ private void ReceivePing(Ping ping)
             {
                 await Task.WhenAll(awaitables);
             }
-            catch (AggregateException e)
+            catch (Exception e)
```

Comment:
@limebell What kind of exceptions do you expect to be thrown here?

Reply:
Usually `TimeoutException`, which occurs when the received peer is not alive actually.

Reply:
I made `TimeoutException` (which is replaced by `PingTimeoutException`) to remove peer from lookup candidate, which will be covered by `SwarmTest.FindSpecificPeerAsyncFail()` test.